### ISSUE TITLE
Document resource imports

### ISSUE THREAD

**Issue by JonasBak** [Unleash/terraform-provider-unleash#91]
### Describe the feature request

It looks like resources support importing:

https://github.com/Unleash/terraform-provider-unleash/blob/fbc8c70c98eeb6aaf09b5ec33af5d3150497acfe/internal/provider/user_resource.go#L104

But this isn't reflected in the docs: https://registry.terraform.io/providers/Unleash/unleash/latest/docs/resources/user

It would be nice to have this available in the docs, so I don't need to check the code to see if it's supported (and so I can see if I can use the user email, or if I should use the user id).

**Comment by thomasheartman** [comment]
Good point! Yeah, that sounds like something that should go in the docs. We've taken note. Thanks for reporting it 🙏🏼
